usermanager.js

```diff
@@ -12,7 +12,13 @@ const roommanager = require("./roommanager");
 const pwd = securePassword();
 const log = getLogger("usermanager");
 const router = express.Router();
-let redisClient;
+const redis = require('redis');
+let redisClient = redis.createClient({
+	port: process.env.REDIS_PORT || undefined,
+	host: process.env.REDIS_HOST || undefined,
+	password: process.env.REDIS_PASSWORD || undefined,
+	db: process.env.REDIS_DB || undefined,
+});
```

Comment:
redisClient gets set in `usermanager.init()`

Reply:
I just test it out and made the container work

Reply:
oh i think i see what's happening.

`redisClient` is undefined when the rate limit objects are created, so rate-limit-redis store defaults to create a new client, which works fine normally, but not when you need to pass in connection options

I'm going to refactor how this works so that it can grab the `redisClient` from a centralized place, instead of having to be told via the `init` method

Reply:
Done! I also merged that into this branch and resolved the merge conflicts.